codespell_lib/_codespell.py

```diff
@@ -401,9 +401,17 @@ def parse_options(args):
 
     # Load config files and look for ``codespell`` options.
     cfg_files = ['setup.cfg', '.codespellrc']
+    tomlfile = 'pyproject.toml'
     if options.config:
         cfg_files.append(options.config)
     config = configparser.ConfigParser()
+
+    # Read toml before other config files.
+    if os.path.isfile(os.path.realpath(tomlfile)):
+        import tomli
+        with open(tomlfile, 'rb') as f:
+            data = tomli.load(f).get('tool', {})
+        config.read_dict(data)
```

Comment:
To actually test this, you can add some new test using a `tmp_path` fixture and add a `pyproject.toml` file in that dir, and run `codespell` from that temporary directory. If it's not clear, let me know and I can push a commit

Reply:
I have read <https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html?highlight=tmp_path> but I still don't know how to realize it. Can you push a commit? Thanks.